commands/kubernetes.go

```diff
@@ -349,15 +360,22 @@ func kubernetesKubeconfig() *Command {
 
 	k8sCmdService := kubernetesCommandService()
 
-	CmdBuilder(cmd, k8sCmdService.RunKubernetesKubeconfigShow, "show <cluster-id|cluster-name>", "Show a Kubernetes cluster's kubeconfig YAML", `
+	cmdShowConfig := CmdBuilder(cmd, k8sCmdService.RunKubernetesKubeconfigShow, "show <cluster-id|cluster-name>", "Show a Kubernetes cluster's kubeconfig YAML", `
 This command prints out the raw YAML for the specified cluster's kubeconfig.	`, Writer, aliasOpt("p", "g"))
+	AddIntFlag(cmdShowConfig, doctl.ArgKubeConfigExpirySeconds, "", 0,
+		"The length of time the cluster credentials will be valid for in seconds. By default, the credentials expire after seven days.")
```

Comment:
The default is a little bit more involved for older cluster versions, but I think we can ignore that (or alternatively leave a pointer to [the relevant docs page](https://www.digitalocean.com/docs/kubernetes/how-to/connect-to-cluster/)).